qiita_db/analysis.py

```diff
@@ -25,7 +25,7 @@
 from biom import load_table
 from biom.util import biom_open
 import pandas as pd
-from skbio.util import find_duplicates
+from skbio.util import flatten
```

Comment:
`flatten` has been removed in later versions of skbio. In other parts of the code, we have been using the construction below to use flatten:
```python
from itertools import chain
list(chain.from_iterable(ITERABLE))
```